src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +12,13 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+    private UserService userService = new UserService();
+
     public boolean login(String email, String password) {
+        User user = userService.findByEmail(email);
+        if (user != null) {
```

Comment:
could you make one line of return instead of 4?

Reply:
@Elena-Bruyako , cool) thanks! fixed